editor/src/clj/editor/collection_non_editable.clj

```diff
@@ -249,12 +250,13 @@
   ;; InstanceDescs, EmbeddedInstanceDescs, and CollectionInstanceDescs.
   ;;
   ;; The resulting resources build targets will be connected to the
-  ;; resource-property-build-targets input of our NonEditableCollectionNode.
+  ;; own-resource-property-build-targets input of our NonEditableCollectionNode.
   ;;
-  ;; Elsewhere, any referenced collection, game object, and component will also
-  ;; have their resource-property-build-targets output connected to our input to
-  ;; ensure we have access to the non-overridden resource property dependencies.
-  ;; As a result, resource-property-build-targets will include not only our own
+  ;; Elsewhere, any referenced collection, game object, and component will have
+  ;; their resource-property-build-targets output connected to our
+  ;; other-resource-property-build-targets input to ensure we have access to the
+  ;; non-overridden resource property dependencies. As a result, our
+  ;; resource-property-build-targets output will include not only our own
```

Comment:
I changed it so that we separate resource-property-build-targets that we've referenced from our collection from those that we "inherit" from referenced component resources, game objects and collections. This lets us cleanly disconnect these in case the resource they refer to by proj-path have moved in the project. Unlike all editable resources, we don't update our internal proj-path references in response to a moved file.